app/assets/stylesheets/base.scss

```diff
@@ -32,22 +32,19 @@
 }
 
 .section {
-  $padding: 10rem;
+  padding: 3rem 0;
 
-  padding-bottom: $padding;
-  padding-top: $padding;
-
-  &.section-gray {
-    background-color: $theme-gray;
+  &.section-blue {
+    background-color: #bfe3ff;
```

Comment:
can extract it into a color variable inside variables.scss

Reply:
done